packages/integration-sdk-testing/src/__tests__/jobState.test.ts

```diff
@@ -63,6 +63,26 @@ describe('entities', () => {
     expect(jobState.collectedEntities).toEqual([]);
   });
 
+  test('getEntity returns entity from jobState when _type and _key match', async () => {
+    expect.hasAssertions();
+    const jobState = createMockJobState();
+    await jobState.addEntities(inputEntities);
+
+    const result = await jobState.getEntity({ _type: 'test_a', _key: 'a' });
+
+    expect(result).toEqual(inputEntities[0]);
+  });
+
+  test('getEntity throws Error when entity cannot be found in jobState', async () => {
+    expect.hasAssertions();
```

Comment:
Do you actually need `expect.hasAssertions()`? If you're awaiting expects with promises, I don't think you should.

Reply:
You are indeed correct. I removed this and the other one which was also unnecessary.